client/termsetting/handlers/categorical.ts

```diff
@@ -118,30 +104,30 @@ export function getHandler(self: CategoricalInstance) {
 
 		async postMain() {
 			const body = self.opts.getBodyParams?.() || {}
-			if (self.term.type == 'condition') {
-				for (const key in self.q) {
-					// detect bar_by_* and value_by_* flags
-					if (key.includes('_by_')) body[key] = self.q[key]
-				}
-			}
+			// if (self.term.type == 'condition') {
```

Comment:
@xzhou82 @siosonel I commented out code related to condition terms rather than removing outright. All tests pass. Not sure how to test for any residual problems. Any suggestions? Is maybe there's another step I should include for testing (?).

Reply:
i did some additional testing with regression UI and it seems to be fine. you can delete